lib/CMakeLists.txt

```diff
@@ -302,6 +302,7 @@ target_link_libraries(libfacter PRIVATE
     ${CMAKE_THREAD_LIBS_INIT}
 )
 
+symbol_exports(libfactersrc "${CMAKE_CURRENT_LIST_DIR}/inc/facter/export.h")
```

Comment:
I'm also not *entirely* certain that's the cause of our problem, but it seems to be something that approximates "GCC is assuming declarations are dllimport, but definitions aren't". I actually can't find `dllimport` or `dllexport` anywhere in facter or leatherman, so I have no idea how our export stuff works on Windows.